cloudnatix/manifest.json

```diff
@@ -0,0 +1,76 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "caebfbe2-48ba-443f-b2c6-bd80122b2605",
+  "app_id": "cloudnatix",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Integration with CloudNatix",
+    "title": "CloudNatix",
+    "media": [
+      {
+        "media_type": "image",
```

Comment:
Please update this image with a new screenshot once our recommended changes are applied to your dashboard. In your  new screenshot, please ensure that the purple "Add Widgets" box is not in the screenshot.

Reply:
Sure, updated the images!

Reply:
Sorry for the back and forth, but all images must have an aspect ratio of 16:9 in order to not get cut off in the media carousel. Can you ensure that the image fits this aspect ratio, either by taking a new screenshot or placing the screenshot on a 16:9 background? 

You can view our [guidelines for images here](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel).